### ISSUE TITLE
Typos and proposals 

### ISSUE THREAD

**Issue by hasebe** [enakai00/colab_jaxbook#1]
### Typos

- All notebooks
  - Note: This notebook is `desinged` -> Note: This notebook is `designed`
- Chapter02/1. Logistic regression model.ipynb
  - [LRM-07] - Generate `inital` parameter values. -> Generate `initial` parameter values.
  - [LRM-10] - jax.`vlue`_and_grad() -> jax.`value`_and_grad()
- Chapter02/3. MNIST softmax estimation.ipynb
  - [MSE-06] - Generate `inital` parameter values. -> Generate `initial` parameter values.
- Chapter03/1. Single layer network example.ipynb
  - [SLN-05] - Generate `inital` parameter values. -> Generate `initial` parameter values.
- Chapter03/2. MNIST single layer network.ipynb
  - [MSL-06] - Generate `inital` parameter values. -> Generate `initial` parameter values.
- Chapter03/3. Hyper parameter tuning.ipynb
  - [HPT-10] - Train models with different number of nodes, and record the `reults`. -> Train models with different number of nodes, and record the `results`.
- Chapter03/4. Double layer network example.ipynb
  - [DLN-05] - Generate `inital` parameter values. -> Generate `initial` parameter values.
- Chapter05/3. CNN interpretation - image generation.ipynb
  - [CII-08] - `Defnine` a dual layer CNN model. -> `Define` a dual layer CNN model.
- Chapter05/5. Transfer Learning with ResNet.ipynb
  - [TLR-23] - Show prediction `reults` for sample images. -> Show prediction `results` for sample images.
- Chapter05/6. MNIST autoencoder example.ipynb
  - [MAE-07] - Define a loss function as the mean `squar` error. -> Define a loss function as the mean `square` error.
- Chapter05/7. MNIST DCGAN example.ipynb 
  - [MDE-08] - Define a loss `funcion` to train the generator. -> Define a loss `function` to train the generator.
  - [MDE-09] - Define a loss `funcion` to train the generator. -> Define a loss `function` to train the generator.

### Need to update to align with source code?

- Chapter01/2. Least squares method with native JAX functions.ipynb
  - [LSJ-04] - (x=`0`, ..., 12) -> (x=`1`, ..., 12)
- Chapter01/3. Least squares method with Flax and Optax.ipynb
  - [LSF-04] - (x=`0`, ..., 12) -> (x=`1`, ..., 12)

**Comment by enakai00** [comment]
Thanks for the thorough check!
